pages/common/pueue-switch.md

```diff
@@ -0,0 +1,8 @@
+# pueue switch
+
+> Switches the queue position of two enqueued or stashed commands.
+> More information: <https://github.com/Nukesor/pueue>.
+
+- Switch the priority of two tasks:
+
+`pueue switch {{task_id}} {{task_id}}`
```

Comment:
I've changed my mind here; it's unclear to just use `task_id` when the context is about switching them.
```suggestion
`pueue switch {{task_id1}} {{task_id2}}`
```